internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -580,10 +580,15 @@ func convertNetworkConfig(network manifest.NetworkConfig) template.NetworkOpts {
 		SubnetsType:    template.PublicSubnetsPlacement,
 		SecurityGroups: network.VPC.SecurityGroups,
 	}
-	if network.VPC.Placement == nil {
+	placement := network.VPC.Placement
+	if placement.IsEmpty() {
 		return opts
 	}
-	if *network.VPC.Placement != manifest.PublicSubnetPlacement {
+	if placement.PlacementString == nil {
+		opts.SubnetIDs = placement.PlacementArgs.Subnets
```

Comment:
do we want to set `opts.AssignPublicIP = template.DisablePublicIP` here? Otherwise it will be set to true

Reply:
Good idea. I'll have a follow-up PR adding the CFN template changes and an integ test